### PR TITLE
add Huawei and Eddystone docs

### PR DESCRIPTION
**Pull request by nickpatrick**

### CODE REVIEW

File: docs/beacons.mdx [hunk 1]
```diff
@@ -106,11 +110,13 @@ The CSV may have the following columns:
 For example, to import beacons installed on registers and in parking spaces at a store, you could use the following CSV:
 
 ```json
-description,tag,external id,type,uuid,major,minor,coordinates,enabled,metadata
-"Store #123 - Register #1",store-register,123-1,ibeacon,"b9407f30-f5f8-466e-aff9-25556b57fe6d",100,1,"[-73.975363,40.783826]",true,{}
-"Store #123 - Register #2",store-register,123-2,ibeacon,"c9407f30-f5f8-466e-aff9-25556B57fe6d",100,1,"[-73.975363,40.783826]",true,{}
-"Store #123 - Parking Space #1",store-parking,123-1,ibeacon,"d9407f30-f5f8-466e-aff9-25556b57fe6d",101,1,"[-73.975363,40.783826]",true,{}
-"Store #123 - Parking Space #2",store-parking,123-2,ibeacon,"e9407f30-f5f8-466e-aff9-25556b57fe6d",101,2,"[-73.975363,40.783826]",true,"{""type"":""parking""}"
+description,tag,externalId,type,uuid,major,minor,uid,instance,coordinates,enabled,metadata
```
Comment (jsani-radar) [inline docs/beacons.mdx:113]: The CSV import schema still says "`ibeacon` is currently supported" (L100)

Screen Shot 2022-08-22 at 8 49 37 PM

File: docs/beacons.mdx [hunk 2]
```diff
@@ -62,17 +66,17 @@ Beacon entry and exit events are sent to the SDK, to [webhooks](/integrations#we
...
 
-By default, Radar will sync nearby beacons to the SDK on every location update using the `coordinates` specified for each beacon. The SDK will then range nearby beacons based on `uuid`, `major`, and `minor`.
+By default, Radar will sync nearby beacons to the SDK on every location update using the `coordinates` specified for each beacon. The SDK will then range nearby beacons based on `uuid`, `major`, and `minor` for iBeacon or `uid` and `instance` for Eddystone.
 
-Alternatively, the SDK can range beacons based on `uuid` only, regardless of the `coordinates` specified for each beacon. To enable this, set _iBeacon UUIDs_ on the [Settings page](https://radar.com/dashboard/settings), under _Beacons_. On iOS, you must also enable the _Location updates_ [background mode](/sdk/ios#background-modes)
+Alternatively, the SDK can range beacons based on `uuid` or `uid` only, regardless of the `coordinates` specified for each beacon. To enable this, set _iBeacon UUIDs_ and/or `Eddystone UIDs` on the [Settings page](https://radar.com/dashboard/settings), under _Beacons_. On iOS, you must also enable the _Location updates_ [background mode](/sdk/ios#background-modes)
```
Comment (jsani-radar) [inline docs/beacons.mdx:71]: nit: missing an ending period

File: docs/beacons.mdx [hunk 3]
```diff
@@ -62,17 +66,17 @@ Beacon entry and exit events are sent to the SDK, to [webhooks](/integrations#we
...
 
 ## Create beacons
 
 You can create beacons via the dashboard, CSV import, or through the API. You can create beacons in the Test environment for development and testing, and in the Live environment for production.
 
 Like [geofences](/geofences), you also specify the metadata for beacons when you create them, including *tag* (a group for the beacon, e.g., `store-register`), external ID (an external ID for the beacon that maps to your internal database, e.g., `123-1`), *description* (a display name for the beacon, e.g., `Store #123 - Register #1`).
 
-You also specify the identifiers used to range the beacon, including `uuid`, `major`, and `minor`, as well as an approximate latitude and longitude used to sync nearby beacons (within 1 kilometer) to the SDK.
+You also specify the identifiers used to range the beacon, including `uuid`, `major`, and `minor` for iBeacon or `uid` and `instance` for Eddystone, as well as an approximate latitude and longitude used to sync nearby beacons (within 1 kilometer) to the SDK.
```
Comment (jsani-radar) [inline docs/beacons.mdx:79]: out of scope for this PR, but wondering if we should have a `Types` section on this page explaining `ibeacon` / `eddystone` and their identifiers

File: docs/beacons.mdx [hunk 4]
```diff
@@ -106,11 +110,13 @@ The CSV may have the following columns:
...
-"Store #123 - Parking Space #2",store-parking,123-2,ibeacon,"e9407f30-f5f8-466e-aff9-25556b57fe6d",101,2,"[-73.975363,40.783826]",true,"{""type"":""parking""}"
+description,tag,externalId,type,uuid,major,minor,uid,instance,coordinates,enabled,metadata
+"Store #123 - Register #1",store-register,123-1,ibeacon,"b9407f30-f5f8-466e-aff9-25556b57fe6d",100,1,,,"[-73.975363,40.783826]",true,{}
+"Store #123 - Register #2",store-register,123-2,ibeacon,"c9407f30-f5f8-466e-aff9-25556B57fe6d",100,2,,,"[-73.975363,40.783826]",true,{}
+"Store #123 - Parking Space #1",store-parking,123-1,ibeacon,"d9407f30-f5f8-466e-aff9-25556b57fe6d",101,1,,,"[-73.975363,40.783826]",true,{}
+"Store #123 - Parking Space #2",store-parking,123-2,ibeacon,"e9407f30-f5f8-466e-aff9-25556b57fe6d",101,2,,,"[-73.975363,40.783826]",true,"{""type"":""parking""}"
+"Store #123 - Aisle A",store-aisle,123-A,eddystone,,,,"f7826da6bc5b71e0893e","4456384b726a","[-73.975363,40.783826]",true,{}
+"Store #123 - Aisle B",store-aisle,123-B,eddystone,,,,"f7826da6bc5b71e0893e","4456384b726b","[-73.975363,40.783826]",true,{}
```
Comment (jsani-radar) [inline docs/beacons.mdx:119]: super nit: let's adjust some of these coordinates? if someone copy / pastes this into import, they'll have 6 overlapping beacons

File: docs/sdk/android.mdx [hunk 5]
```diff
@@ -50,9 +50,11 @@ You can also add the SDK to your project manually. Download [the current release
 
 ## Dependencies
 
-The SDK depends on [AndroidX](https://developer.android.com/jetpack/androidx) and Play Services Location version `17.1.0` and higher. These libraries will be automatically included as transitive dependencies by Gradle. Learn more about managing dependencies in Gradle [here](https://docs.gradle.org/current/userguide/introduction_dependency_management.html#sec:dependency_resolution).
+The SDK depends on [AndroidX](https://developer.android.com/jetpack/androidx) and [Google Play Services Location](https://developer.android.com/training/location) version `17.1.0` and higher. These libraries will be automatically included as transitive dependencies by Gradle. Learn more about managing dependencies in Gradle [here](https://docs.gradle.org/current/userguide/introduction_dependency_management.html#sec:dependency_resolution).
```
Comment (jsani-radar) [inline docs/sdk/android.mdx:53]: nit: doesn't seem like the Gradle link exists as I'm getting a 404, this seems like the replacement - https://docs.gradle.org/current/userguide/core_dependency_management.html

**Review by jsani-radar — COMMENTED** [review]
looking good overall - wondering if we want to leave the `eddystone` rows out of import for a follow-on PR

**Comment by nickpatrick** [comment]
@jsani-radar Great feedback, will address tomorrow

**Comment by nickpatrick** [comment]
@jsani-radar Good to 👍 and merge?

**Review by jsani-radar — APPROVED** [review]

**Pull request merged**